tq42/experiment_run.py

```diff
@@ -70,6 +71,52 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return f"ExperimentRun: {MessageToJson(self.data, preserving_proto_field_name=True)}"
 
+    @property
+    def completed(self) -> bool:
```

Comment:
should this maybe be `done` and we can check against `completed` and `failed`? 🤔

Reply:
I'm using this for both `result` and `outputs` which are only defined if the run is completed and not failed.

Reply:
Yeah meant it more from a user perspective so that it is easy to know whether the run is finished (completed or failed) but we can add this also later.